### PR TITLE
Voting on review implemented

### PR DESCRIPTION
**Pull request by Szpytma**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import axios from "axios";
+import { async } from "q";
```
Comment (alexdevdavis) [inline src/api.js]: was this an auto-add? can it be deleted

File: src/api.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 
 const baseUrl = "https://nc-games-rkbx.onrender.com/api/";
```
Comment (alexdevdavis) [inline src/api.js]: ideally you'll use `axios.create()` to create an axios instance and set the baseURL; in production, this allows you to set headers and such for authentication, authorisation and suchlike.

File: src/components/Review.jsx [hunk 3]
```diff
@@ -8,12 +8,19 @@ function Review() {
...
   const [isLoading, setIsLoading] = useState(true);
+  const [disabled, setDisabled] = useState(false);
 
   const formatDate = (dataString) => {
     const date = new Date(dataString);
     const formattedDate = date.toLocaleString();
     return formattedDate;
   };
+  const handleClick = () => {
+    api.patchReviewVotes(review_id).then((updatedReview) => {
```
Comment (alexdevdavis) [inline src/components/Review.jsx]: using optimistic rendering will improve UX
- removing the delay
- ensuring that the given like is accounted for, as opposed to including concurrent likes from elsewhere

**Review by alexdevdavis — APPROVED** [review]
Nicely done, the site feels good to use.
- note comment on optimistic rendering; strongly suggest you implement this before moving on

File: src/api.js (hunk 1 above)
Comment (Szpytma) [reply src/api.js]: I have no idea where it is coming from - as I deleted it twice already. It should be sorted with the next push

**Pull request merged**